Dockerfile

```
@@ -1,5 +1,5 @@
-# Override this based on the architecture; this is currently pointing to amd64
-ARG BASE_SHA="fce37e5146419a158c2199c6089fa39b92445fb2e66dc0331f8591891239ea3b"
+# Override this based on the architecture; this is currently pointing to amd64; may be set empty to use latest platform specific base
+ARG BASE_SHA="@sha256:fce37e5146419a158c2199c6089fa39b92445fb2e66dc0331f8591891239ea3b"
```

```suggestion
ARG BASE_DIGEST="sha256:e7fe469c4e729ff0ed6ff464f41eaff0e4cb9b6fe7efe71754d8935c8118eb87"
ARG BASE_SHA="${BASE_SHA:-@$BASE_DIGEST}"
```

This needs to be updated to the latest pinned digest we use, and to avoid having the `@` in the label, we'll use two variables. That way, you still can build using `--build-arg BUILD_SHA=` which remove any pinning, and the label value will still be correct. This is very hacky though, but it should work for now. We will add ARM support for Docker soon, so at that point I'll try to figure out a better solution.

This means that lower, where we set the base digest label (line 36), you need to replace the variable `BASE_SHA` with `BASE_DIGEST`.